Var/Makefile

```diff
@@ -0,0 +1,14 @@
+CC = gcc
```

Comment:
The recommended way of doing the makefile is to have 4 directories:
`source/` (or `src/`) for all the `.c` files and other sources
`include/` for `.h` files and other headers.
`build/` for `.o` files and other binary files used only when compiling.
`bin/` for the final results of compiling - usually the binary program to run.

Reply:
But if the .h files and .c files aren't in the same folder then how do the .c files with some header inclusion know how find the .h files?

Reply:
1. `man gcc`, has the flag `-I` for that.
2. It's unnecessary to delete them and compiling without them would take much more time. However, that doesn't mean you should keep them when moving a project. They are only used when building (hence `build/`) and are not part of the final product.

P.S: When checking the program to make sure everything works before submitting it, it's recommended to erase the `build/` directory each time in-order to guarantee there weren't any changes in the code that did not make the final binary.